src/app/loadout-builder/process/useProcess.ts

```diff
@@ -222,20 +228,27 @@ const groupComparator = chainComparator(
 );
 
 /**
- * This groups items for process depending on whether any general, other or raid mods are locked as follows
- * - If there are general, other or raid mods locked it groups items by (stats, masterworked, modSlot, energyType).
- * - If there are only general mods locked it groupes items by (stats, masterwork, energyType)
- * - If no general, other or raid mods are locked it groups by (stats, masterworked).
+ * To reduce the number of items sent to the web worker we group items by a number of varying
+ * parameters, depending on what mods and armour upgrades are selected.
+ *
+ * After items have been grouped we only send a single item (the first one) as a representitive of
+ * said group. All other grouped items will be available by the swap icon in the UI.
  *
- * Note that assumedMasterwork effects this.
+ * It can group by any number of the following concepts depending on locked mods and armor upgrades,
+ * - Stat distribution
+ * - Masterwork status
```

Comment:
What's the concept behind grouping by masterwork? Wouldn't they just have better stats (unless you've asked to assume masterworked stats)? For something like class items, why would we want to send a masterworked and an un-masterworked item?

Reply:
I think you are right. Initially I believe, it was a way to handle class items before we mapped items to `ProcessItems` and got the derived stat values with assume masterwork settings but now it is probably not needed. I will remove it and see what happens.